appveyor.yml

```diff
@@ -30,11 +30,6 @@ environment:
     - TARGET_ARCH: x64
       CONDA_PY: 34
 
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
```

Comment:
That is correct. We no longer have `gdal` on `PY35+Win`. @gillins did a nice job with geos. I guess `gdal` is next :wink: